### ISSUE TITLE
wrong priority parsing

### ISSUE THREAD

**Issue by francogp** [icamys/php-sitemap-generator#17]
I added 
```
generator->addURL($loc, $lastModified, 'monthly', 0.9);
```
but it generates:
``` 
<url>
    <loc>https://test-web.com</loc>
    <lastmod>2020-08-25T08:09:12-03:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0,9</priority>
</url>
```
and that priority violates the rules for for float numbers
(note that my php is configured for south america localizations and numbers)

**Comment by icamys** [comment]
@francogp Thanks for your comment! At the moment you can just pass string instead of float to the `addURL` method:

```
generator->addURL($loc, $lastModified, 'monthly', '0.9'); // 0.9 -> '0.9'
```

**Comment by francogp** [comment]
> @francogp Thanks for your comment! At the moment you can just pass string instead of float to the `addURL` method:
> 
> ```
> generator->addURL($loc, $lastModified, 'monthly', '0.9'); // 0.9 -> '0.9'
> ```

No, it does the same issue

**Comment by icamys** [comment]
@francogp It's odd as the priority should be stored as a string and no changes should be applied. Could you please provide the output for this command? I need a bit more info to debug the problem.

```
$ php -r 'phpinfo();' | grep "'LC_"
```

**Comment by francogp** [comment]
> @francogp It's odd as the priority should be stored as a string and no changes should be applied. Could you please provide the output for this command? I need a bit more info to debug the problem.
> 
> ```
> $ php -r 'phpinfo();' | grep "'LC_"
> ```

![image](https://user-images.githubusercontent.com/2739706/92478837-23cb6f00-f1b9-11ea-95d4-be2b4bb8edea.png)

**Comment by icamys** [comment]
@francogp Great! I'll try to reproduce the bug when I'll have some spare time. Thanks for the assistance.

**Comment by icamys** [comment]
@francogp I've fixed the issue in version 2.0.3. Please check it out.

**Comment by francogp** [comment]
> @francogp I've fixed the issue in version 2.0.3. Please check it out.

Yep.. seems to be fixed. Thanks
